CONTRIBUTING.md

```diff
@@ -48,6 +48,26 @@ If several `It` sections are similar, consider refactoring them in a `DescribeTa
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
 - [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet) - Common resources for existing developers
 
+## Managing releases
+
+### OperatorHub.io
+
+After a new version is released, it must be added manually to the
+[k8s-operatorhub/community-operators](https://github.com/k8s-operatorhub/community-operators) repo to be available on
+OperatorHub.io.
+
+First, generate all manifests:
+
+```shell
+make operator-hub-release IMAGE_TAG=<RELEASE_TAG> VERSION=<VERSION_NUMBER>
```

Comment:
```suggestion
make operatorhub-release IMAGE_TAG=<RELEASE_TAG> VERSION=<VERSION_NUMBER>
```

Using `operator-hub` is confusing with the `kernel-module-management-operator-hub` terminology IMO.